### PR TITLE
Initial version check + proxy start script

### PR DESCRIPTION
**Pull request by tegola**
WIP.
The only things working are the version check and the `start-proxy` script.

Oh, and `brew cask` isn't used anymore, just `brew install`.

### CODE REVIEW

File: bin/install.macosx [hunk 1]
```diff
@@ -43,7 +43,7 @@ then
   brew install wget curl
 
   # Install virtualbox.
-  brew cask install virtualbox
+  brew install virtualbox
```
Comment (paolomainardi) [inline bin/install.macosx:46]: this one cannot be executed anymore on big sur (especially on M1)

File: config/macosx/start-proxy.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+echo "Now starting a new dinghy-http-proxy container..."
+docker run -d --restart=always \
+  -v /var/run/docker.sock:/tmp/docker.sock:ro \
+  -v ~/.dinghy/certs:/etc/nginx/certs \
+  -p 80:80 -p 443:443 -p 19322:19322/udp \
+  -e DNS_IP=127.0.0.1 -e CONTAINER_NAME=http-proxy -e DOMAIN_TLD=loc \
+  --name http-proxy \
+  codekitchen/dinghy-http-proxy
```
Comment (paolomainardi) [inline config/macosx/start-proxy.sh:8]: This file should be placed under a bin directory as we do for Ubuntu: https://github.com/sparkfabrik/sparkdock/tree/master/config/ubuntu/bin

File: ansible/macos/macos11.yml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+          force: yes
+
+      - name: Test DNSDOCK - Run dummy container
+        shell: eval "$(dinghy env)" && docker run -d -e DNSDOCK_ALIAS=test1.redis.docker.loc --name redis-test redis:alpine
+
+      - name: Test DNSDOCK - Run dummy container
+        shell: ping -c1 test1.redis.docker.loc
+
+      - name: Test DNSDOCK - Run dummy container
+        shell: eval "$(dinghy env)" && docker kill redis-test && docker rm redis-test
```
Comment (paolomainardi) [inline ansible/macos/macos11.yml:185]: This file for big sur for now it should empty or just copy the the binary, like this one: https://github.com/sparkfabrik/sparkdock/blob/master/ansible/ubuntu/ubuntu20.yml#L70

File: bin/install.macosx (hunk 1 above)
Comment (tegola) [reply bin/install.macosx:46]: `brew cask` is deprecated, but `brew install` works fine for installing `cask`-ed software on Big Sur (Intel, no idea on M1).

**Pull request merged**